### PR TITLE
Update to liberty-maven-plugin-2.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by josephca**

### CODE REVIEW

**Comment by josephca** [comment]
@patricktiu : can you provide code review? Thanks,

File: README.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ This sample can be installed onto runtime versions 8.5.5.0 and later.
...
 2. Right-click the project and select **Run As > Maven Clean**.
 3. Right-click the project and select **Run As > Maven Install**.
-4. Right-click the project and select **Run As > Maven Build...** then run the goal `liberty:run-server`.
+4. Right-click the project and select **Run As > Run on Server**.
 5. You should see the following in the console: `Application PhoneBookBySpringMVC started in XX.XX seconds.`
 
 ## Running with Maven
 
-This project can be built with [Apache Maven](http://maven.apache.org/). The project uses [Liberty Maven Plug-in][] to automatically download and install Liberty profile runtime from the [Liberty repository](https://developer.ibm.com/wasdev/downloads/). Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server. 
+This project can be built with [Apache Maven](http://maven.apache.org/). The project uses [Liberty Maven Plug-in][] to automatically download and install Liberty with Java EE7 Web Profile runtime from the [Maven Central](https://developer.ibm.com/wasdev/downloads/). Liberty Maven Plug-in is also used to create, configure, and run the application on the Liberty server.
```
Comment (patricktiu) [inline README.md]: The URL for Maven Central is incorrect.  I suggest you to remove the link.

File: pom.xml [hunk 2]
```diff
@@ -263,56 +188,6 @@
 						<appLocation>${project.artifactId}.war</appLocation>
```
Comment (patricktiu) [inline pom.xml]: I prefer to use ${project.build.finalName}.war

Comment (josephca) [inline pom.xml]: Fixed.

File: README.md (hunk 1 above)
Comment (josephca) [inline README.md]: Removed the link.

**Comment by josephca** [comment]
Found an issue (**not a regression**) while running 'mvn liberty:run-server'. The url (http://localhost:9811/phonebook/) with both version of maven plugins works fine though. We may need to handle this issue separately. 

[err] log4j:WARN No appenders could be found for logger (org.springframework.web.context.ContextLoader).
[err] log4j:WARN Please initialize the log4j system properly.
[err] log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.

**Comment by patricktiu** [comment]
Please try the original sample if the problem exists, open an issue against sample.

**Comment by josephca** [comment]
The same issue is reproducible with original sample. Opened an issue: https://github.com/WASdev/sample.Spring/issues/2

**Pull request merged**